warmup/statistics.py

```diff
@@ -10,7 +10,7 @@ def mean(seq):
     return math.fsum(seq) / len(seq)
 
 
-def bootstrap_confidence_interval(seq, confidence=CONFIDENCE):
+def bootstrap_confidence_interval(seq, confidence=CONFIDENCE, iterations=ITERATIONS):
```

Comment:
Adding a kwarg won't affect the existing call-sites. The default is used. I only added this so that I could test with 1 iteration to make testing fast.